src/tests/conformance/broadcast/mod.rs

```diff
@@ -0,0 +1,4 @@
+//! Contains tests for cases when a message is expected after a handshake.
```

Comment:
are all of these messages actual broadcasts? otherwise a `post_handshake` module would be a more fitting term

Reply:
> otherwise a post_handshake module would be a more fitting term
Aha, you are right. I think `TmValidation` and `TmProposeLedgedrs` are plain broadcast messages, we seem to receive them periodically.
And `TmManifests` and `TmEndpoints` are received just a few times after a handshake.

Reply:
Should we then have both the `broadcast` and the `post _handshake` modules?

Reply:
`post_handshake` could be a parent module for all kinds of messages applicable then, including broadcasts.

Reply:
Okay, then something like this:

- tests
  - conformance 
    - handshake.rs
    - post_handshake
      - query/...
      - cmd/...
      - broadcast/...

Reply:
The reorganization will be done in a later PR.